src/Entity/BannerSelection/Bucket.php

```diff
@@ -10,30 +10,35 @@
 class Bucket {
 
 	private $identifier;
+	private $mainBanner;
 
 	/**
 	 * @var Banner[]
 	 */
 	private $banners = [];
 
-	public function __construct( string $identifier, Banner $mainBanner, Banner ...$otherBanners ) {
+	public function __construct( string $identifier, Banner $mainBanner, array $otherBanners ) {
```

Comment:
I removed it because `$buckets` held the main banner in index 0 and then in index 1 there was an array of other banners. I thought that's not right and made it "flat" array this way.